generators/client/templates/angular/src/main/webapp/app/admin/admin.module.ts.ejs

```diff
@@ -18,8 +18,14 @@
 -%>
 import { NgModule, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
 import { RouterModule } from '@angular/router';
+import { HttpClient } from '@angular/common/http';
+import { TranslateModule, TranslateLoader, MissingTranslationHandler } from '@ngx-translate/core';
+import { TranslateHttpLoader } from '@ngx-translate/http-loader';
+import { JhiLanguageService, JhiMissingTranslationHandler } from 'ng-jhipster';
+import { JhiConfigService } from 'ng-jhipster/src/config.service';
```

Comment:
why qualified import? import { JhiConfigService } from 'ng-jhipster` should work

Reply:
I believe 'ng-jhipster' was not exporting that from index.js

Reply:
i'll look into it